hasher-matcher-actioner/hmalib/common/messages/banks_api.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import typing as t
+from dataclasses import dataclass
+
+from hmalib.lambdas.api.middleware import JSONifiable
+from hmalib.common.models.bank import Bank
+
+
+@dataclass
+class AllBanksEnvelope(JSONifiable):
```

Comment:
nit: it feels like this could stay in `hasher-matcher-actioner/hmalib/lambdas/api/bank.py` as we've done with most others; messages seemed to me primarily to be for lambda to lambda via sqs or sns. 

super nit: we've used `response` instead of `envelope` don't have a strong preference in naming as long as we can trend towards consistency though

Reply:
I had a different reason for moving this here, but unless a lot more work is done, that does not make sense. You're right, I'll move it to api/bank.py instead.

I wanted the API representations in a separate module that could be made available to `hmaclient` should that ever come to life. Also, for my gencode experiments. But we can tackle both later. For now, moving back and renaming Envelope -> Response.